scripts/testCardTypes.js

```diff
@@ -104,6 +104,12 @@ const testAnkiCardCreation = async typeDir => {
   return result;
 };
 
+const testPackageInterface = async typeDir => {
+  const type = typeDir.replace('types/', '');
```

Comment:
I can fell how it's really annoying but I can't run this code without any errors if it works on windows 😭.
It works correctly if I replace this string to this one
`  const type = typeDir.replace("types\\", "");`

Reply:
I'll fix it )

Reply:
@zenby  this one should work

Reply:
You're right! Thanks.